make-xges.py

```diff
@@ -27,11 +27,57 @@ def file_to_uri(path):
     path = os.path.realpath(path)
     return 'file://' + path
 
+def my_time_format(raw_time):
+
+    # total amount of seconds
+    seconds = 0.0
+
+    if raw_time is None:
```

Comment:
If we're calling the parser from argparse, then I don't think we'd need this case: the conversion function won't be called if the default value is not a string.

Reply:
Without that condition:

```
./make-xges.py --start=1:00 dir/ file.xges
Traceback (most recent call last):
  File "./make-xges.py", line 468, in <module>
    sys.exit(main(sys.argv))
  File "./make-xges.py", line 463, in main
    p = Presentation(opts)
  File "./make-xges.py", line 73, in __init__
    self.end   = my_time_format( opts.end )
  File "./make-xges.py", line 41, in my_time_format
    parts = raw_time.split(':')
AttributeError: 'NoneType' object has no attribute 'split'
```

Reply:
My suggestion was to drop all the changes to methods on `Presentation`, so the conversion from time string to float happens while parsing arguments.

This way, you'd never be asked to convert the `None` value.